doc/DataPlaneCodeGeneration/DeveloperDrivenEvolution.md

```diff
@@ -109,6 +130,7 @@ namespace Azure.AI.MetricsAdvisor
     {
         public virtual async Task<Response> GetMetricFeedbackAsync(Guid feedbackId, RequestContext context = null)
         {
+            using var scope = ClientDiagnostics.CreateScope("MetricsAdvisorClient.GetMetricFeedback");
```

Comment:
Please indicate there are other logic in this method instead of just 1 line of code.